context/app/static/js/components/SampleDetail.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import Typography from '@material-ui/core/Typography';
 import ProvTabs from './Detail/ProvTabs';
 import Summary from './Detail/Summary';
-import Attribution from './Detail/Attribution';
+import Attribution from './Detail/Attribution/Attribution';
```

Comment:
Export from an index.js? I worry about tying ourselves too closely to a particular file structure.

Reply:
Should be fixed in #410.